helloextend-protection/includes/class-helloextend-protection-cart-offer.php

```
@@ -76,106 +74,137 @@ public function hooks()
-    // goes through the cart and gets updates to products/plans for normalization
-    public function get_cart_updates()
+    private function is_item_helloextend($item)
+    {
+        $warranty_product_id = $this->settings['warranty_product_id'];
+        return $item['product_id'] == $warranty_product_id && isset($item['extendData']) && !empty($item['extendData']);
+    }
+
+    private function is_lead($item)
+    {
+        return $this->is_item_helloextend($item) && $item['extendData']['leadToken'] && $item['extendData']['leadQuantity'];
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'is_lead'. (Unused Code Rules)

(UnusedPrivateMethod)